jenkins/Jenkinsfile

```diff
@@ -345,9 +347,12 @@ pipeline {
     stage('Init git submodules') {
         when { expression { !shouldAbortBuild } }
         steps {
-            sh """               
-                git submodule update --init --recursive                
-            """
+            sshagent(['jenkins-ssh-github']) {
```

Comment:
huh... odd that this is needed now, before it was only required for the github write permissions it granted when pushing to a repo

Reply:
Yeah -- it's a private repo so it needs it to read as well. Probably don't want our mining detection rules to be readable to the world.